sql/search_derived/search_clients_daily_v8/metadata.yaml

```diff
@@ -6,3 +6,5 @@ labels:
   schedule: daily
 scheduling:
   dag_name: bqetl_search
+  allow_field_addition_dates:
+  - 2020-07-13
```

Comment:
> It would be helpful to see an example use of this functionality included in this PR.

done

Reply:
Can you also run dag generation and commit the results?

Reply:
done